<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #1C1C1C,
    stop:0.5 #333333,
    stop:1 #4C4C4C);
	background-origin: content;
	background-clip: content;
}

QLineEdit {
	border: 1px solid #555555;
    border-radius: 4px;
    padding: 2px;
    background-color: #333333;
    color: #FFFFFF;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>481</width>
      <height>86</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#ffffff;&quot;&gt;AUTOMATIZADOR DE&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#ff5500;&quot;&gt; PDF &lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#ffffff;&quot;&gt;Y &lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#00aa00;&quot;&gt;EXCEL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>467</y>
      <width>481</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-weight: bold;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>91</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pdf1_1Button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: red;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: #8B0000;
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pdf1_2Button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: red;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: #8B0000;
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pdf2_1Button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: red;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: #8B0000;
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pdf2_2Button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: red;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: #8B0000;
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="excelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: green;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: rgb(0, 85, 0);
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pathButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: blue;
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: rgb(0, 0, 127);
}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>100</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditPdf1_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>PDF 1: SEMESTRE 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPdf1_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>PDF 1: SEMESTRE 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPdf2_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>PDF 2: SEMESTRE 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPdf2_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>PDF 2: SEMESTRE 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditExcel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>EXCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSave">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>RUTA DE GUARDADO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditYear1">
       <property name="placeholderText">
        <string>AÃ±o 1</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditYear2">
       <property name="placeholderText">
        <string>AÃ±o 2</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autoButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: rgb(85, 0, 127);
    border: none;
    border-radius : 10px;
    color: rgb(255, 255, 255);
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: rgb(85, 0, 255);
}</string>
       </property>
       <property name="text">
        <string>AUTOMATIZAR</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="creditos">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>380</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;Desarrollado por: Julio Y Eduardo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>60</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: yellow;
    border: none;
    border-radius : 12px;
    color: black;
	font-weight: bold;
	padding: 5px;
}

QPushButton::hover {
	background-color: rgb(186, 186, 93);
}</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <action name="actionInformaci_n_General">
   <property name="text">
    <string>InformaciÃ³n General</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
